inc/auth.php

```diff
@@ -160,6 +159,53 @@ function auth_loadACL() {
     return $out;
 }
 
+/**
+ * Try a token login
+ *
+ * @return bool true if token login succeeded
+ */
+function auth_tokenlogin() {
+    global $USERINFO;
+    global $INPUT;
+    /** @var DokuWiki_Auth_Plugin $auth */
+    global $auth;
+    if(!$auth) return false;
+
+    // see if header has token
+    $header = '';
+    if(function_exists('apache_request_headers')) {
+        // Authorization headers are not in $_SERVER for mod_php
+        $headers = apache_request_headers();
+        if(isset($headers['Authorization'])) $header = $headers['Authorization'];
```

Comment:
with apache 2.4 the header is lowercased: `$headers['authorization']`

Reply:
but not all keys are lowercased, for example

`'Host' => 'localhost:8087'`

Reply:
```suggestion
        if(isset($headers['Authorization'])) $header = $headers['Authorization'];
        if(!$header && isset($headers['authorization'])) $header = $headers['authorization'];
```

Reply:
I pushed an update that makes sure the header is lower case